vnext/ReactUWP/Utils/XamlDirectInstance.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <winrt/Windows.UI.Xaml.Core.Direct.h>
```

Comment:
We need to figure out consuming the nuget package for xaml direct to keep the down-level support.

I had tried adding the nuget before, but the winrt headers where not getting  auto-generated.
Is the pattern to just check in the generated headers? for nuget packages?

Reply:
My commit where I added the dependency on MUX is here: https://github.com/jonthysell/react-native-windows/commit/0f9024e80d28620dacf862cbebfa8c2f38ac8a88

There I have the command need to run to generate the headers.

Unfortunately, our current model would probably be to check them in.

Reply:
cool! thank you